components/status_dropdown/status_dropdown.tsx

```diff
@@ -63,35 +69,99 @@ export default class StatusDropdown extends React.PureComponent <Props, State> {
         },
     }
 
-    isUserOutOfOffice = (): boolean => {
-        return this.props.status === UserStatuses.OUT_OF_OFFICE;
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+            openUp: false,
+            width: 0,
+        };
+    }
+
+    getCurrentDateTime = (tz: UserTimezone, enable: boolean): Date => {
```

Comment:
Is there any specific reason this is a member function? Looks like we could probably make this a helper function and put it with the other utils.